Detail popup

estebanmual opened the pull request:
In this milestone we: 
- Make sure that there are no linter errors.
- Make sure that you used the correct GitHub flow.
- Implement the following interactions: 
   -  When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
   - When the user clicks (or taps) the close (X) button, the popup disappears.
- Implement popups for both mobile and desktop screen sizes 
- Generate the information dynamically using an object array

estebanmual committed:
Give dynamic display to projects section

VicPeralta committed:
Create popup skeleton

new file:   details.html
modified:   script.js

estebanmual committed:
Add style to the detail popup

mmsesay reviewed (changes requested):
Hi @Estebanmual , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Changing the name of portfolio title to your name

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mmsesay commented:
- [ ] I love the way you implemented the pop window. But I noticed has some responsive issues. It overflows on the desktop version and doesn't show the entire content of the selected project. Can you please take a look into it?

Your current modal popup window |
:-------------------------:|
![sh](https://user-images.githubusercontent.com/39523312/151202874-5dca9a8b-d684-491f-8eb5-5e41da72cc6e.gif)

- [ ] Looking at your live version, I was unable to actually click on a project to see its details. I think the hovering effect is affecting it. It will be nice if you could please take a look into it also.

Your current live version |
:-------------------------:|
![sh2](https://user-images.githubusercontent.com/39523312/151203496-ef219f7a-d1a8-4414-90a9-ccde72a1e499.gif)

estebanmual committed:
Popup changes

estebanmual committed:
Add scrollable to details pop up

VicPeralta committed:
Fix linter errors. No linter errors

deleted:    00_Website/assets/css/styles-home-page-desktop.css
modified:   00_Website/assets/css/styles-home-page.css
modified:   package-lock.json

mateo951 reviewed (approved):
### Hi Team, 

Nice work making the changes requested by the previous reviewer 😎
Your project is complete!  There is nothing else to say other than... it's time to merge it ✔
### Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
